index.html

```diff
@@ -10,7 +10,51 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <header>
+        <h1>Weather Report</h1>
+        <span>
+            Location:
+            <span id="cityLocation" class="city">Seattle</span>
+        </span>
+    </header>
+    <section class="temp" id="temp">
+        <h2>Temperature</h2>
+        <section class="temp_control_section">
+            <span id="topButton">⏫</span>
+            <span id="tempValue" class="orange">65</span>
+            <span id="bottomButton">⏬</span>
```

Comment:
```suggestion
            <button id="topButton">⏫</button>
            <p id="tempValue" class="orange">65</p>
            <button id="bottomButton">⏬</button>
```
Some small changes to use tags that have semantic meaning